src/main/resources/findologic_20.xsd

```diff
@@ -21,10 +21,16 @@
                     </xs:unique>
                 </xs:element>
             </xs:sequence>
-            <xs:attribute type="xs:string" name="version"/>
+            <xs:attribute type="versionType" name="version" use="required"/>
```

Comment:
Docu of old version already says that version is required.
![image](https://user-images.githubusercontent.com/2993034/195065506-a30adb12-7d16-4504-850d-286b8436fb92.png)

Reply:
I am not sure if we have to reflect the old docs on the new version, please check it again during review!

Reply:
> I am not sure if we have to reflect the old docs on the new version

We don't have to do that in general, but i think having the version available in the XML file could make life easier as Homie or similar. So we know which version is actually used by only checking the raw file. Will keep this discussion open til review

Reply:
I've removed the change until we got more input of PO. I've added this issue to [FINDO-9598](https://findologic.myjetbrains.com/youtrack/issue/FINDO-9598) Open topics XML schema 
so that we do not forget it.